### ISSUE TITLE
[Bug]: SSplotComparisons - issue with input weight-at-age vector

### ISSUE THREAD

**Issue by Lee-Cronin-Fine-NOAA** [r4ss/r4ss#969]
### Describe the bug

I can't compare a model with an input weight-at-age vector to a model that does not have an input weight-at-age vector. 

The only difference between the models is that the first line of in the ctl file. One model reads in the wtatage.ss file while the other does not. 

### To Reproduce

use SS_output() to read the result from both models.

run SSplotComparisons()

### Expected behavior

I expected comparison plots comparing the models to be produced

### Screenshots

![SSplotComparisons-issue](https://github.com/user-attachments/assets/aa61cd2b-afd2-4e5b-8897-9a78b302b29a)

### Which OS are you seeing the problem on?

Windows

### Which versions of r4ss and SS3 are you seeing the problem on?

1.49.2

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/r4ss/r4ss#code-of-conduct)

**Comment by iantaylor-NOAA** [comment]
Thanks for reporting this @Lee-Cronin-Fine-NOAA and sorry for both the error and the slow response while I was in the field. In 3fc6feb I've just pushed a fix to the main branch that should correct your error. 

I'm seeing a separate errors when the wtatage model has a very different range of years than the parametric growth model. Both are associated with the relatively recent addition (via https://github.com/r4ss/r4ss/pull/966) of the summary biomass to the set of values being compared. I'll will continue to work on the other issue but didn't want you and others to have to wait longer than necessary for the most important fix. Let me know if you continue to have issues.

**Comment by Lee-Cronin-Fine-NOAA** [comment]
Hey Ian,

I have updated r4ss and I am able to run the SSplotComparisons() function. There is one issue I have noticed and it has to do with the Age-0 recruitment plot. The plot shows recruitment for the model with fixed weight-at-age (Base-3, in first attached figure) is significantly higher than the other models (millions version thousands). 
![compare9_recruits](https://github.com/user-attachments/assets/01924789-e73c-47f8-9e94-f83679731c10)

However, when I look at the r4ss age-0 recruitment plot for just the Base-3 model the recruitment scale is in the thousand (second attached figure). I believe there is a scaling error in SSplotComparisons() function. 

![ts11_Age-0_recruits_(1000s)](https://github.com/user-attachments/assets/9daa0589-ff44-4b0f-9ef5-9a1ad1782c62)
